### PR TITLE
Add an error message on-screen if all or a single movie can't load du…

### PR DESCRIPTION
**Pull request by justenanderson-commits**
____ Wrote Tests ____ Implemented ____ Reviewed

Necessary checkmarks:

[x] All Tests are Passing

[x] The code will run locally
Type of change

[x] New feature
[] Bug Fix
Implements/Fixes:
Adds error message for network errors. 

description closes #
Check the correct boxes

[x] This broke nothing
[] This broke some stuff
[] This broke everything
Testing Changes

[] No Tests have been changed
[] Some Tests have been changed
[] All of the Tests have been changed(Please describe what in the world happened)
Checklist:

[x] My code has no unused/commented out code
[x] I have reviewed my code
[] I have commented my code, particularly in hard-to-understand areas
[x] I have fully tested my code

### CODE REVIEW

**Review by dgubko — APPROVED** [review]
Thank you for working on it ! looks great!

File: src/components/AllMovies.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ class AllMovies extends React.Component {
       <div className="section--all-movies">
         <section className="heading--all-movies">
           <h2>Movies</h2>
+          <h2>{ this.state.error }</h2>
```
Comment (sertmer) [inline src/components/AllMovies.js:30]: This H2 on line 30 is going to be an empty string most of the time which could be really confusing to anyone using a screen reader. Better to only show this if the Error is there. Remember that && Syntax? Might be good to refactor :)

File: src/components/SingleMovie.js [hunk 2]
```diff
@@ -28,9 +28,9 @@ class SingleMovie extends React.Component {
 
   render() {
     return (
-      this.state.movie && (
+      this.state.movie ? (
         <div className="single-page-container">
-          <img src={this.state.movie.poster_path} />
+          <img src={this.state.movie.poster_path} alt="Movie poster image"/>
```
Comment (sertmer) [inline src/components/SingleMovie.js:33]: So good to see you're still thinking about accessibility with that alt tag!

File: src/components/SingleMovie.js [hunk 3]
```diff
@@ -28,9 +28,9 @@ class SingleMovie extends React.Component {
...
           <div className="single-movie-description">
             <h2>
               {this.state.movie.title} (
```
Comment (sertmer) [inline src/components/SingleMovie.js:36]: You've got a few `this.state.movie._____`. Might be a good use case for destructuring!

**Review by sertmer — COMMENTED** [review]
Left a few comments and questions - check them out and let me know if you have a need for clarification!

**Pull request merged**